addons/payment_worldline/models/payment_provider.py

```diff
@@ -66,13 +67,15 @@ def _worldline_make_request(self, endpoint, payload=None, method='POST'):
         content_type = 'application/json; charset=utf-8' if method == 'POST' else ''
         tz = datetime.timezone(datetime.timedelta(hours=0), 'GMT')
         dt = datetime.datetime.now(tz).strftime('%a, %d %b %Y %H:%M:%S %Z')  # Datetime in RFC1123.
-        signature = self._worldline_calculate_signature(method, endpoint, content_type, dt)
+        signature = self._worldline_calculate_signature(method, endpoint, content_type, dt, idempotency_key)
```

Comment:
Nit:  following our [coding style](https://www.odoo.com/odoo/knowledge/8790) we try to respect 100 chars :)
```suggestion
        signature = self._worldline_calculate_signature(
            method, endpoint, content_type, dt, idempotency_key
        )
```

Reply:
Yes. Sorry! I knew about it :eyes: Gonna config my editor to highlights it :wink: Will be easier. Thanks.